README.md

```diff
@@ -752,34 +752,34 @@ cromwell-executions/
 └── three_step
     └── a59651fc-4d9a-4fed-99ba-f5e2c9d84bb4
         ├── call-cgrep
-        │   ├── Users
-        │   │   └── jdoe
-        │   │       └── projects
-        │   │           └── cromwell
-        │   │               └── cromwell-executions
-        │   │                   └── three_step
-        │   │                       └── a59651fc-4d9a-4fed-99ba-f5e2c9d84bb4
-        │   │                           └── call-ps
-        │   │                               └── stdout
-        │   ├── rc
-        │   ├── script
-        │   ├── stderr
-        │   └── stdout
+        │   ├── Users
```

Comment:
it doesn't appear to be impactful but i'm curious what led to these changes?

Reply:
I used online editor to modify README, then copy/paste, that's why you see this.